### PR TITLE
chore: update seeds for v21 release

### PR DESCRIPTION
**Pull request by knst**
## Issue being fixed or feature implemented
https://github.com/dashpay/dash/issues/6081

## What was done?
Extra onion seeds are provided by pasta.

```sh
dash-cli protx list valid 1 2110129 > protx_list.json

# Make sure the onion seeds still work!
while IFS= read -r line
do
  address=$(echo $line | cut -d':' -f1)
  port=$(echo $line | cut -d':' -f2)
  nc -v -x 127.0.0.1:9050 -z $address $port
done < "onion_seeds.txt"

python3 makeseeds.py protx_list.json onion_seeds.txt > nodes_main.txt
python3 generate-seeds.py . > ../../src/chainparamsseeds.h
```

## How Has This Been Tested?
n/a

## Breaking Changes
n/a

## Checklist:
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: contrib/seeds/README.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ Utility to generate the seeds.txt list that is compiled into the client
 
 The seeds compiled into the release are created from the current protx list, like this:
 
-```bash
-dash-cli protx list valid 1 2018966 > protx_list.json
+```sh
+dash-cli protx list valid 1 > protx_list.json
```
Comment (PastaPastaPasta) [inline contrib/seeds/README.md:9]: you should be specifying specific block here?

Comment (knst) [reply contrib/seeds/README.md:9]: nope, by default it's "current tip"

Comment (PastaPastaPasta) [reply contrib/seeds/README.md:9]: Yeah... Arguably we should be specifying a specific hash though so it's more reproducible

**Review by PastaPastaPasta — DISMISSED** [review]
utACK 133fc20cb384dd2d027baed53d7ab011bf084e07

File: contrib/seeds/README.md (hunk 1 above)
Comment (UdjinM6) [reply contrib/seeds/README.md:9]: You have to pick a specific block to make it reproducible. Could change that to accept block hashes but specifying block height is good enough if that block is chainlocked already.

Comment (PastaPastaPasta) [reply contrib/seeds/README.md:9]: Idk, it's probably fine. We've had trouble actually reproducing these before

Comment (UdjinM6) [reply contrib/seeds/README.md:9]: why make it worse though? :)

**Review by UdjinM6 — CHANGES_REQUESTED** [review]
Should revert changes in `contrib/seeds/README.md` imo, they are both irrelevant and incorrect

**Comment by PastaPastaPasta** [comment]
Sooo what was the height you used? :)

**Comment by knst** [comment]
> dash-cli protx list valid 1 2110129 > protx_list.json

**Comment by PastaPastaPasta** [comment]
Unable to reproduce, I believe due to local issue:
```
'Resolver' object has no attribute 'resolve'
```

File: contrib/seeds/README.md [hunk 2]
```diff
@@ -5,8 +5,8 @@ Utility to generate the seeds.txt list that is compiled into the client
 
 The seeds compiled into the release are created from the current protx list, like this:
 
-```bash
-dash-cli protx list valid 1 2018966 > protx_list.json
+```sh
+dash-cli protx list valid 1 _TIP_ > protx_list.json
```
Comment (UdjinM6) [inline contrib/seeds/README.md:9]: The note above is not accurate too actually, it must be a list from a "safe" block, not a "current" one. I'd still prefer to revert these changes and fix it in a separate PR.

**Comment by UdjinM6** [comment]
I'm getting lots of `ERR: Could not resolve ASN...` so yeah, it's not reproducible for me too 😞

**Comment by knst** [comment]
> I'm getting lots of ERR: Could not resolve ASN... so yeah, it's not reproducible for me too 😞
I got 698 failures:
```
$ cat | grep 'Could not resolve' -c
698
```

File: contrib/seeds/README.md (hunk 2 above)
Comment (knst) [reply contrib/seeds/README.md:9]: I doesn't like hard-coded `2018966` because it's easy to copy-paste and use same `2018966` for next time.

Comment (UdjinM6) [reply contrib/seeds/README.md:9]: I agree but this PR is a wrong place to fix it

**Review by UdjinM6 — APPROVED** [review]
utACK 43c39537d898008d5cd28425ea80acf5ee5133ac

**Review by PastaPastaPasta — APPROVED** [review]
utACK https://github.com/dashpay/dash/commit/43c39537d898008d5cd28425ea80acf5ee5133ac
